packages/core/src/interpreter.ts

```diff
@@ -765,6 +765,9 @@ export class Interpreter<
           } else {
             this.logger(value);
           }
+        },
+        [actionTypes.assign]: () => {
+          /* No-op to prevent no implementation warning */
```

Comment:
this should be flagged with a TODO or something as it's a strong indication that things are wired weirdly here